packages/xterm.rb

```diff
@@ -3,46 +3,83 @@
   })
   binary_sha256({
-    aarch64: 'e61f8a1bfc06e700ec42b0d3f7e97b9b1e0f8120a33fbfc6efeba3eabda24126',
-     armv7l: 'e61f8a1bfc06e700ec42b0d3f7e97b9b1e0f8120a33fbfc6efeba3eabda24126',
-       i686: '7912500763cca0d85c1f6d84ec4cce6cbd5b4c9a70090dfe49cc94a395b7b84b',
-     x86_64: 'c6dd0a0eebbe9a84af571c79ad1ecb925b1a16333bb6e75280f5818c86faedb5'
+    aarch64: 'a8302b75325cda9acbf59929ecaf791281e2710b1c9e128354844145aa9c2d40',
+     armv7l: 'a8302b75325cda9acbf59929ecaf791281e2710b1c9e128354844145aa9c2d40',
+       i686: '235f5938788fbc494bac5bac66a15259795ddad186b081ace5a21092f995942c',
+     x86_64: '88353d7c42d3a91c5d7f1440d78808c6625cbf5cb9a186ebe10a8a2caeda37af'
   })
 
-  depends_on 'pcre'
+  depends_on 'libxaw' => :build
   depends_on 'sommelier'
+  depends_on 'freetype' # R
+  depends_on 'glibc' # R
+  depends_on 'harfbuzz' # R
+  depends_on 'libice' # R
+  depends_on 'libutempter' # R
+  depends_on 'libx11' # R
+  depends_on 'libxaw' # R
+  depends_on 'libxext' # R
+  depends_on 'libxft' # R
+  depends_on 'libxinerama' # R
+  depends_on 'libxmu' # R
+  depends_on 'libxpm' # R
+  depends_on 'libxt' # R
+  depends_on 'luit' => :build
+  depends_on 'ncurses' # R
+  depends_on 'pcre' # R
+
+  no_env_options
 
   def self.build
-    system './configure',
-           '--with-x',
-           '--with-pcre',
-           '--with-xinerama',
-           '--enable-dabbrev',
-           '--enable-toolbar',
-           '--with-pkg-config',
-           '--enable-exec-xterm',
-           '--enable-16bit-chars',
-           '--enable-dec-locator',
-           '--enable-double-buffer',
-           '--enable-readline-mouse',
-           '--enable-regis-graphics',
-           '--enable-sixel-graphics',
-           "--prefix=#{CREW_PREFIX}",
-           "--with-xpm=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    system "./configure #{CREW_OPTIONS} \
+      --disable-imake \
+      --enable-16bit-chars \
+      --enable-256-color \
+      --enable-88-color \
+      --enable-ansi-color \
+      --enable-broken-osc \
+      --enable-broken-st \
+      --enable-dabbrev \
+      --enable-dec-locator \
+      --enable-double-buffer \
+      --enable-doublechars \
+      --enable-exec-xterm \
+      --enable-freetype \
+      --enable-i18n \
+      --enable-load-vt-fonts \
+      --enable-logging \
+      --enable-luit \
+      --enable-mini-luit \
+      --enable-narrowproto \
+      --enable-readline-mouse \
+      --enable-regis-graphics \
+      --enable-sixel-graphics \
+      --enable-tcap-query \
+      --enable-toolbar \
+      --enable-warnings \
+      --enable-wide-chars \
+      --with-app-defaults=#{CREW_PREFIX}/share/X11/app-defaults/ \
+      --with-pcre \
+      --with-pkg-config \
+      --with-tty-group=tty \
+      --with-utempter \
+      --with-x \
+      --with-xinerama \
+      --with-xpm=#{CREW_PREFIX}"
+    system "sed -i '/^LDFLAGS/ s/$/ -DUSE_TERMINFO=1 -ltinfo/' Makefile"
```

Comment:
Move to `def self.patch` section?

Reply:
Can't, as configure needs to run first.

But I'm having a discussion upstream about how to do this cleanly...

Reply:
Can it be patched in `Makefile.in` prior to the `configure`?

Reply:
Possibly... but I'm waiting for an upstream fix to resolve the issue without needing patching...

I've sent my config.status and config.log to the maintainer as requested...

Reply:
I just went ahead and added a patch section to modify the Makefile.in...